.vscode/settings.json

```diff
@@ -10,5 +10,9 @@
     "typescript.tsc.autoDetect": "off",
     "i18n-ally.localesPaths": [
         "languages"
-    ]
+    ],
+    "files.autoSave": "onFocusChange",
+    "github.copilot.enable": {
+        "*": true,
```

Comment:
Sind das nicht die lokalen Settings von dir? Sollten wir dann nicht besser den kompletten Ordner .vscode in die gitignore machen?